Ert workflow test failing

```sh
FAILED tests/test_ert_integration/test_hook_implementations.py::test_hook_implementations - AssertionError: assert set() == {'WF_COPY_PRE...ASE_METADATA'}
  
  Extra items in the right set:
  'WF_CREATE_CASE_METADATA'
  'WF_COPY_PREPROCESSED_DATAIO'
  
  Full diff:
  + set()
  - {
  -     'WF_COPY_PREPROCESSED_DATAIO',
  -     'WF_CREATE_CASE_METADATA',
  - }
FAILED tests/test_ert_integration/test_hook_implementations.py::test_hook_implementations_docs - AssertionError: assert {'WF_COPY_PRE...ASE_METADATA'} == set()
  
  Extra items in the left set:
  'WF_CREATE_CASE_METADATA'
  'WF_COPY_PREPROCESSED_DATAIO'
  
  Full diff:
  - set()
  + {
  +     'WF_COPY_PREPROCESSED_DATAIO',
  +     'WF_CREATE_CASE_METADATA',
  + }
============ 2 failed, 455 passed, 1 xfailed, 46 warnings in 43.79s ============
```

--- a/src/fmu/dataio/hook_implementations/jobs.py
+++ b/src/fmu/dataio/hook_implementations/jobs.py
@@ -1,10 +1,8 @@
 from __future__ import annotations
 
-from ert import plugin
-from ert.plugins.plugin_manager import hook_implementation
+import ert
 
 
-@hook_implementation
-@plugin(name="fmu_dataio")
+@ert.plugin(name="fmu_dataio")
 def installable_workflow_jobs() -> dict:
     return {}

--- a/src/fmu/dataio/scripts/copy_preprocessed.py
+++ b/src/fmu/dataio/scripts/copy_preprocessed.py
@@ -12,19 +12,11 @@ import argparse
 import logging
 import warnings
 from pathlib import Path
-from typing import TYPE_CHECKING, Final
+from typing import Final
 
-from ert.plugins.plugin_manager import hook_implementation
+import ert
 
 from fmu.dataio import ExportPreprocessedData
-
-try:
-    from ert import ErtScript
-except ImportError:
-    from res.job_queue import ErtScript
-
-if TYPE_CHECKING:
-    from ert.plugins.workflow_config import WorkflowConfigs
 
 logger: Final = logging.getLogger(__name__)
 
@@ -62,7 +54,7 @@ def main() -> None:
     copy_preprocessed_data_main(commandline_args)
 
 
-class WfCopyPreprocessedData(ErtScript):
+class WfCopyPreprocessedData(ert.ErtScript):
     """A class with a run() function that can be registered as an ERT plugin.
 
     This is used for the ERT workflow context. It is prefixed 'Wf' to avoid a
@@ -147,8 +139,8 @@ def get_parser() -> argparse.ArgumentParser:
     return parser
 
 
-@hook_implementation
-def legacy_ertscript_workflow(config: WorkflowConfigs) -> None:
+@ert.plugin(name="fmu_dataio")
+def legacy_ertscript_workflow(config: ert.WorkflowConfigs) -> None:
     """Hook the WfCopyPreprocessedData class with documentation into ERT."""
     workflow = config.add_workflow(
         WfCopyPreprocessedData, "WF_COPY_PREPROCESSED_DATAIO"

--- a/src/fmu/dataio/scripts/create_case_metadata.py
+++ b/src/fmu/dataio/scripts/create_case_metadata.py
@@ -13,20 +13,12 @@ from __future__ import annotations
 import argparse
 import logging
 from pathlib import Path
-from typing import TYPE_CHECKING, Final
+from typing import Final
 
+import ert
 import yaml
-from ert.plugins.plugin_manager import hook_implementation
 
 from fmu.dataio import CreateCaseMetadata
-
-try:
-    from ert import ErtScript
-except ImportError:
-    from res.job_queue import ErtScript
-
-if TYPE_CHECKING:
-    from ert.plugins.workflow_config import WorkflowConfigs
 
 logger: Final = logging.getLogger(__name__)
 logger.setLevel(logging.CRITICAL)
@@ -66,7 +58,7 @@ def main() -> None:
     create_case_metadata_main(commandline_args)
 
 
-class WfCreateCaseMetadata(ErtScript):
+class WfCreateCaseMetadata(ert.ErtScript):
     """A class with a run() function that can be registered as an ERT plugin.
 
     This is used for the ERT workflow context. It is prefixed 'Wf' to avoid a
@@ -162,8 +154,8 @@ def get_parser() -> argparse.ArgumentParser:
     return parser
 
 
-@hook_implementation
-def legacy_ertscript_workflow(config: WorkflowConfigs) -> None:
+@ert.plugin(name="fmu_dataio")
+def legacy_ertscript_workflow(config: ert.WorkflowConfigs) -> None:
     """Hook the WfCreateCaseMetadata class with documentation into ERT."""
     workflow = config.add_workflow(WfCreateCaseMetadata, "WF_CREATE_CASE_METADATA")
     workflow.parser = get_parser

--- a/tests/test_ert_integration/test_hook_implementations.py
+++ b/tests/test_ert_integration/test_hook_implementations.py
@@ -1,4 +1,4 @@
-import os
+from __future__ import annotations
 
 from ert.plugins.plugin_manager import ErtPluginManager
 
@@ -6,7 +6,7 @@ import fmu.dataio.hook_implementations.jobs
 from fmu.dataio.scripts import copy_preprocessed, create_case_metadata
 
 
-def test_hook_implementations():
+def test_hook_implementations() -> None:
     plugin_manager = ErtPluginManager(
         plugins=[
             fmu.dataio.hook_implementations.jobs,
@@ -15,20 +15,19 @@ def test_hook_implementations():
         ]
     )
 
-    expected_forward_models = set()
+    expected_forward_models: set[str] = set()
     installable_fms = plugin_manager.get_installable_jobs()
     assert set(installable_fms) == expected_forward_models
 
     expected_workflow_jobs = {"WF_CREATE_CASE_METADATA", "WF_COPY_PREPROCESSED_DATAIO"}
-    installable_workflow_jobs = plugin_manager.get_installable_workflow_jobs()
-    for wf_name, wf_location in installable_workflow_jobs.items():
+    installable_workflow_jobs = plugin_manager.get_ertscript_workflows().get_workflows()
+    for wf_name, _ in installable_workflow_jobs.items():
         assert wf_name in expected_workflow_jobs
-        assert os.path.isfile(wf_location)
 
     assert set(installable_workflow_jobs) == expected_workflow_jobs
 
 
-def test_hook_implementations_docs():
+def test_hook_implementations_docs() -> None:
     plugin_manager = ErtPluginManager(
         plugins=[
             fmu.dataio.hook_implementations.jobs,
@@ -45,7 +44,7 @@ def test_hook_implementations_docs():
         assert fm_docs[fm_name]["examples"] != ""
         assert fm_docs[fm_name]["category"] != "other"
 
-    installable_workflow_jobs = plugin_manager.get_installable_workflow_jobs()
+    installable_workflow_jobs = plugin_manager.get_ertscript_workflows().get_workflows()
     wf_docs = plugin_manager.get_documentation_for_workflows()
     assert set(wf_docs) == set(installable_workflow_jobs)
     for wf_name in installable_workflow_jobs:

